clientconn.go

```diff
@@ -1274,6 +1274,9 @@ func (ac *addrConn) createTransport(addr resolver.Address, copts transport.Conne
 
 	newTr, err := transport.NewClientTransport(connectCtx, ac.cc.ctx, addr, copts, onGoAway, onClose)
 	if err != nil {
+		if logger.V(2) {
+			logger.Errorf("NewClientTransport failed to connect to %s. Err: %v", addr, err)
```

Comment:
I find that it is better to avoid filler words like "failed" etc in error strings and it leads to better/concise error strings. This could be written as:

`logger.Errorf("NewClientTransport(%q): %v", addr, err)`

or if you prefer ,ore human readable names over function names

`logger.Errorf("Creating new client transport to %q: %v", addr, err)`